### PR TITLE
Updates doc 'what why and how' for prose/clarity

### PR DESCRIPTION
**Pull request by thewonderfx**
Some changes to the prose of the document, to increase readability and clarity.

### CODE REVIEW

File: content/docs/01-what-why-and-how.md [hunk 1]
```diff
@@ -29,40 +29,42 @@ These are the things that WebRTC will give you. This list is not exhaustive, jus
...
-This is a topic that takes an entire book to explain. However, to start off we break it into four steps.
+The WebRTC Protocol is an immense topic that would take an entire book of its own to explain in full. We can, however, begin an explanation by breaking it into four sequential steps:
 
 * Signaling
 * Connecting
 * Securing
 * Communicating
 
-These four steps happen sequentially. The prior step must be 100% successful for the subsequent one to even begin.
+As these steps happen sequentially, the prior step must be 100% successful for the subsequent step to even begin.
```
Comment (ClaytonMcCray) [inline content/docs/01-what-why-and-how.md:43]: What about

"These steps are sequential: the prior step must be 100% successful for the subsequent step to begin."

File: content/docs/01-what-why-and-how.md [hunk 2]
```diff
@@ -29,40 +29,42 @@ These are the things that WebRTC will give you. This list is not exhaustive, jus
...
 
-One peculiar fact about WebRTC is that each step is actually made up of many other protocols! To make WebRTC, we stitch together many existing technologies. In that sense, WebRTC is more a combination and configuration of well-understood tech that has been around since the early 2000s.
+One peculiar fact about WebRTC is that each step is actually made up of many other protocols; to make WebRTC, we stitch together many existing technologies. In that sense, you can think of WebRTC as being more a combination and configuration of well-understood tech dating back to the early 2000s than as a brand-new process in its own right.
 
-Each of these steps has dedicated chapters, but it is helpful to understand them at a high level first. Since they depend on each other, it will help when explaining further the purpose of each of these steps.
+Each of these steps has its own dedicated chapter but, since each step depends on the one before it, it is helpful to understand them at a high level first. A basic understand of all the steps will help when explaining further the purpose of each individually.
```
Comment (ClaytonMcCray) [inline content/docs/01-what-why-and-how.md:47]: "Each of these steps has its own dedicated chapter"

This changes the original text semantically. Is there only one chapter per step? The original does not imply that.

"A basic understand" -> "A basic understanding"

File: content/docs/01-what-why-and-how.md (hunk 1 above)
Comment (thewonderfx) [reply content/docs/01-what-why-and-how.md]: I removed the 'sequential' from the preceding sentence and numbered the list.  As suggested it almost felt like we were defining 'sequential' twice.

File: content/docs/01-what-why-and-how.md (hunk 2 above)
Comment (thewonderfx) [reply content/docs/01-what-why-and-how.md:47]: They do each have their own chapters, though Communicating is split into multiple, which I'd like to try to merge eventually.

File: content/docs/01-what-why-and-how.md [hunk 3]
```diff
@@ -29,40 +29,42 @@ These are the things that WebRTC will give you. This list is not exhaustive, jus
 * Congestion control
 * Sub-second latency
 
+This list is not meant to be exhaustive, but rather a range of things you may appreciate during your journey. If you don't know all these terms yet, don't worry: this book will teach them to you along the way.
+
 ## The WebRTC Protocol is a collection of other technologies
 
-This is a topic that takes an entire book to explain. However, to start off we break it into four steps.
+The WebRTC Protocol is an immense topic that would take an entire book of its own to explain in full. We can, however, begin an explanation by breaking it into four steps:
```
Comment (ansorren) [inline content/docs/01-what-why-and-how.md:36]: I know hackers love a challenge, but maybe the word `immense` is a bit too discouraging?

**Comment by mogren** [comment]
Rebased into #294

**Pull request closed without merging**